homeassistant/components/irm_kmi/data.py

```diff
@@ -0,0 +1,16 @@
+"""Define data classes for the IRM KMI integration."""
+
+from typing import Any
+
+from irm_kmi_api import CurrentWeatherData, ExtendedForecast
+
+from homeassistant.components.weather import Forecast
+
+
+class ProcessedCoordinatorData(dict[str, Any]):
+    """Data class that will be exposed to the entities consuming data from an IrmKmiCoordinator."""
+
+    current_weather: CurrentWeatherData
+    hourly_forecast: list[Forecast] | None
+    daily_forecast: list[ExtendedForecast] | None
+    country: str
```

Comment:
why does it extend a `dict`? Should we make this a dataclass?

Reply:
By extending `dict` it inherits all the methods of `dict`, like `get()`.  I turned this into a dataclass with `get()` implemented in 8aa7d2ab122